anki_addon/ui/word_info.py

```diff
@@ -307,6 +307,11 @@ def fill_note(self, note):
                 if examples:
                     note['Examples'] = '<br>'.join(f"{ex}" for ex in examples)
 
+            if lexeme.tags:
+                rection = [tag for tag in lexeme.tags if tag is not None]
```

Comment:
If I'm not mistaken, I think rection is not the only thing that ends up in the tags list, so I think it should be parsed separately and stored as a separate field in the `Lexeme` dataclass.

Btw, can `tag` element be `None`? If yes, I will filter it out in `Sonaveeb` during parsing. There's no reason for `Lexeme.tags` list to contain `None`-s. But let's not worry about it in this PR, I've started working on #5, and it will probably take more time than this, so keeping the diff minimal here would be helpful for me during conflicts resolution.